mlrun/db/httpdb.py

```diff
@@ -55,10 +55,23 @@ def bool2str(val):
     return "yes" if val else "no"
 
 
+HTTP_RETRY_AMOUNT = 3
```

Comment:
```suggestion
HTTP_RETRY_COUNT = 3
```

I prefer COUNT here, but won't start a war over it...